docs/man/man1/xhc-hb04.1

```diff
@@ -108,91 +100,83 @@ Example:
 
 .SH Input Pins (Control)
 .TP
-(bit in) \fIxhc\-hb04.stepsize\-up\fR A 1 pulse on this pin changes the
-stepsize to the next higher stepsize in the stepsize sequence specified
-in the xhc\-hb04 (loadusr) command.
+(bit in) \fIxhc\-hb04.stepsize\-up\fR
+A 1 pulse on this pin changes the stepsize to the next higher stepsize in the stepsize sequence specified in the xhc\-hb04 (loadusr) command.
 .TP
-(bit in) \fIxhc\-hb04.stepsize\-down\fR A 1 pulse on this pin changes the
-stepsize to the next lower stepsize in the stepsize sequence specified
-in the xhc\-hb04 (loadusr) command.
+(bit in) \fIxhc\-hb04.stepsize\-down\fR
+A 1 pulse on this pin changes the stepsize to the next lower stepsize in the stepsize sequence specified in the xhc\-hb04 (loadusr) command.
 
 .SH Input Pins (to the pendant LCD display)
 .TP
-(float in) \fIxhc\-hb04.[xyza].pos\-absolute\fR Absolute position display.
-(typically connect to: halui.axis.N.pos\-feedback). The LCD display
-for pos\-absolute is fixed format with a sign, 4 number digits and 3
-fraction digits (+XXXX.XXX), require: \-9999.999 <= value <= 9999.999.
+(float in) \fIxhc\-hb04.[xyza].pos\-absolute\fR
+Absolute position display.  (typically connect to: halui.axis.N.pos\-feedback).
```

Comment:
```suggestion
(float in) \fIxhc\-hb04.[xyza].pos\-absolute\fR Absolute position display. 
(typically connect to: halui.axis.N.pos\-feedback).
```

Reply:
I just checked, the line breaks indeed have an effect on how the text is rendered. IMHO the pin names are too difficult to read when the description starts right afterward, so I decided for the line break after the pin. The current version is in 2.9 is inconsistent on this already.